dvc/path/local.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from dvc.path import BasePathInfo, Schemes
+
+
+class LocalPathInfo(BasePathInfo):
+    scheme = Schemes.LOCAL
+
+    def __init__(self, path, url=None):
+        super(LocalPathInfo, self).__init__(url=url, path=path)
```

Comment:
Looks like this passthrough is not needed. Or are you doing that because for some PathInfos url and path are required? They are actually required for all PathInfos, so might make sense to make BasePathInfo.__init__ accept (url, path) not as kwargs.